docs/Features.md

```diff
@@ -0,0 +1,29 @@
+# Features
+
+## XML Tag Auto Close
+
+[vscode-xml](https://github.com/redhat-developer/vscode-xml) provides an `auto close` feature for XML tags:
+
+Using `/` in an opening tag will auto close the tag.
+
+![Auto Close Tag 1](images/Features/AutoCloseTag1.gif)
```

Comment:
I'd probably rename the file and reference to AutoCloseTagSlash.gif The other one can be AutoCloseTagEnd.gif. It just makes it easier to know which ones which without even looking at the document.

Reply:
Could you confirm what the DTD generation feature is?

Reply:
Sure. 

- Copy something like https://git.eclipse.org/c/orbit/orbit-recipes.git/plain/apache/ant/org.apache.ant_1.10.10/src/eclipse/ip_log.xml locally, and open it in a folder.
- On the root element you should see a faint (informational) marker. If you select it, or activate quick fix you should see a few actions. I would demo "Generate ... and bind with xml-model"

Reply:
See https://developers.redhat.com/blog/2020/07/08/improved-xml-grammar-binding-and-more-in-red-hat-vs-code-xml-extension-0-13-0